contract_sale_invoicing/tests/test_contract_sale_invoicing.py

```diff
@@ -1,4 +1,5 @@
 # Copyright 2018 Tecnativa - Carlos Dauden
+# Copyright 2021 initos Gmbh
```

Comment:
You didn't change the test code, so you don't deserve authorship. Please remove this line